ui/app/mainui/adaptors/ContactsModelAdaptor.qml

```diff
@@ -0,0 +1,78 @@
+import QtQml 2.15
+
+import StatusQ.Core.Utils 0.1
+import utils 1.0
+
+import SortFilterProxyModel 0.2
+
+QObject {
+    id: root
+
+    /**
+      Expected model structure:
+
+        pubKey                   [string] - unique identifier of a member, e.g "0x3234235"
+        displayName              [string] - member's chosen name
+        preferredDisplayName     [string] - calculated member name according to priorities (eg: nickname has higher priority)
+        ensName                  [string] - member's ENS name
+        isEnsVerified            [bool]   - whether the ENS name was verified on chain
+        localNickname            [string] - local nickname set by the current user
+        alias                    [string] - generated 3 word name
+        icon                     [string] - thumbnail image of the user
+        colorId                  [string] - generated color ID for the user's profile
+        colorHash                [string] - generated color hash for the user's profile
+        onlineStatus             [int]    - the online status of the member
+        isContact                [bool]   - whether the user is a mutual contact or not
+        isVerified               [bool]   - wheter the user has been marked as verified or not
+        isUntrustworthy          [bool]   - wheter the user has been marked as untrustworthy or not
+        isBlocked                [bool]   - whether the user has been blocked or not
+        contactRequest           [int]    - state of the contact request that was sent
+        isCurrentUser            [bool]   - whether the contact is actually ourselves
+        lastUpdated              [int64]  - clock of when last the contact was updated
+        lastUpdatedLocally       [int64]  - clock of when last the contact was updated locally
```

Comment:
I think these 2 timestamps are (or should be!) `double`

Reply:
hmm, looks like just a standard timestamp, and it's int also on nim side.

Reply:
Yeah, it might be an `int64` on NIM side which is fine; but it QML it must be a `double` in order not to overflow

Reply:
So at this stage it seems to be ok, the potential problem may happen when assigning to qml's `int` but it's not the case here. Am I missing sth here?